plugins/reportal/src/azkaban/reportal/util/Reportal.java

```diff
@@ -262,8 +266,8 @@ public void createZipAndUpload(ProjectManager projectManager, User user,
 
       // Populate the job file
       ReportalTypeManager.createJobAndFiles(this, jobFile, jobName,
-          query.title, query.type, query.script, dependentJob, reportalUser,
-          null);
+        query.title, query.type, query.script, dependentJob, reportalUser,
```

Comment:
Nit: why the whitespace changes? These should be indented by 4 spaces (2 indents) since they are wrapped lines. Same elsewhere.